controllers/scheduleController.js

```diff
@@ -3,8 +3,35 @@ const { getSchedule } = require('../utils')
 
 exports.getRoomSchedule = async (req, res) => {
   try {
-    return res.status(HttpStatus.OK).json(req.query.r)
-  } catch (error) {}
+    const orar = await getSchedule('./data/schedule.json')
+    if (req.query.r === undefined) {
+      return res.status(HttpStatus.OK).send(orar) // ya gettin all the rooms, man
+    }
+    const rooms = req.query.r.split(',')
```

Comment:
we should handle the case where r is just a single room and then rooms will be just a string.